docs/reference/auditbeat/auditbeat-installation-configuration.md

```diff
@@ -329,6 +330,7 @@ PS C:\Program Files\auditbeat> Start-Service auditbeat
 
 The default location where Windows log files are stored varies:
 * {applies_to}`stack: ga 9.1` `C:\Program Files\Auditbeat-Data\Logs`
+* {applies_to}`stack: ga 9.0.6` `C:\Program Files\Auditbeat-Data\Logs`
 * {applies_to}`stack: ga 9.0` `C:\ProgramData\auditbeat\Logs`
```

Comment:
I _think_just replacing `9.1` with `9.0.6` would suffice in the current model because the tooltip on the 9.0.6 badge will say: 

>Available on Elastic Stack version 9.0.6 and later unless otherwise specified.

```suggestion
* {applies_to}`stack: ga 9.0.6` `C:\Program Files\Auditbeat-Data\Logs`
* {applies_to}`stack: ga 9.0` `C:\ProgramData\auditbeat\Logs`
```

cc @bmorelli25 to keep me honest...